src/tests/fake/groups.ts

```diff
@@ -14,3 +17,27 @@ export function createFakeGroupMinimal(props?: createFakeGroupMinimalProps): Gro
 
     return merge(defaultGroupMinimal, props);
 }
+
+type createFakeGroupProps = {
+    name?: string;
+    permissions?: Permissions;
+    users?: UserNested[];
+};
+
+export function createFakeGroup(props?: createFakeGroupProps): Group {
+    const { name, permissions, users } = props || {};
+    return {
+        id: faker.random.alphaNumeric(8),
+        name: name || `${faker.random.word()}_group`,
+        permissions: createFakePermissions(permissions),
+        users: users || [],
+    };
+}
+
+export function mockGetGroupAPI(group: Group) {
```

Comment:
I think it might be faster to grok if you standardized on `mockApiGetGroup`. Then we know it is for API mocking from the start instead of having to read to the end of the variable name.

```suggestion
export function mockGetGroupApi(group: Group) {
```